coq-elpi.opam

```diff
@@ -18,8 +18,8 @@ depends: [
   "dune" {>= "3.13"}
   "ocaml" {>= "4.10.0"}
   "elpi" {>= "2.0.7" & < "2.1.0~"}
-  ("coq" {>= "8.20+rc1" & < "8.21~"}
-  | "rocq-core" {>= "9.0+rc1" & < "9.1~"} & "coq-core")
+  ("coq-core" {>= "8.20+rc1" & < "8.21~" | = "dev" } & "coq-stdlib"
+  | "rocq-core" {>= "9.0+rc1" & < "9.1~" | = "dev" } & "coq-core")
```

Comment:
@gares the first dev looks wrong to me. I would expect dev to be possible only in the second case (>= 9.0+rc1)

Reply:
right, indeed that was the case giving me troubles.